application/app/javascript/controllers/inline_field_submit_controller.js

```diff
@@ -11,7 +11,7 @@ export default class extends Controller {
     }
 
     connect() {
-        this.formTarget.classList.add("d-none")
+        //this.formTarget.classList.add("d-none")
```

Comment:
why this?

Reply:
I was checking why was there. This is not needed. The form target should be hidden with CSS initially that should be the contract. This was added initially as a safe guard, but will create a flicker effect if not hidden by default.

So I am removing to make sure the display none is added to the form in the ERB.

I think I should delete the connect method